### ISSUE TITLE
Question: Frame Size

### ISSUE THREAD

**Issue by SimonG4ELI** [lucianodato/libspecbleach#42]
Hi,

I see the frame size is 20ms. If latency is not a problem, is there an advantage in using a much bigger frame size, for example 160ms?

**Comment by lucianodato** [comment]
Right, so yeah that's an option if you want better resolution in the lower frequencies at the expense of more latency. Evidently you already know how to change it. I think here would be a good use of exposing these parameters through a config file

**Comment by SimonG4ELI** [comment]
How about an extra parameter in the specbleach_adaptive_initialize call?

**Comment by lucianodato** [comment]
Yeah I kinda make sense just for this one. In the future I plan to use a multiresolution fft which would get rid of this issue. Will depend on how efficient my implementation is though

**Comment by SimonG4ELI** [comment]
Sounds interesting...

**Comment by lucianodato** [comment]
Do you mind if I change the issue name so it's clear it's a request? To something like Expose frame size configuring possibly through the api

**Comment by SimonG4ELI** [comment]
Sure, rename it as you wish

**Comment by SimonG4ELI** [comment]
> Right, so yeah that's an option if you want better resolution in the lower frequencies at the expense of more latency. Evidently you already know how to change it. I think here would be a good use of exposing these parameters through a config file

In my software's filtering I use FFT Partitioning - I'm sure we could use this to reduce latency, for example I can reduce the latency of 128ms down to 4ms.

**Comment by lucianodato** [comment]
Ahh yeah someone el se has mentioned this once. I'll investigate further, this will be for sure useful!

**Comment by SimonG4ELI** [comment]
> Ahh yeah someone el se has mentioned this once. I'll investigate further, this will be for sure useful!

Uses more CPU but at these sample rates will not be an issue.

**Comment by lucianodato** [comment]
This is actually not compatible with STFT but for time convolution with FIR filters which is not how FFT is used in libspecbleach
